test/cli/data/gapic/api/artman_pubsub.yaml

```diff
@@ -17,37 +17,56 @@ common:
   gapic_api_yaml:
     - ${REPOROOT}/googleapis/google/pubsub/v1/pubsub_gapic.yaml
   output_dir: ${REPOROOT}/artman/output
-  git_repo:
-    # FIXME: Change into the main repo after done with testing.
-    location: git@github.com:lukesneeringer/api-client-staging.git
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   enable_batch_generation: True
-java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-pubsub
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-pubvub-v1
-python:
-  gapic_code_dir: ${REPOROOT}/artman/output/gapic-google-cloud-pubsub-v1
-  git_repo:
-    gapic_subpath: generated/python/gapic-google-cloud-pubsub-v1
-    grpc_subpath: generated/python/proto-google-cloud-pubsub-v1
-go:
-  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
-  git_repo:
-    gapic_subpath: generated/go/google-pubsub-v1
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-pubsub
+go:
+  gapic_code_dir: ${REPOROOT}/gapi-pubsub-go
+  git_repos:
+    go:
+      gapic_subpath: pubsub/apiv1
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
```

Comment:
It would feel more natural for `location` to be before `gapic_subpath`.

Also, `location` is quite generic - maybe `remote_url` would be a better fit.